lpd/tests/test_qualitative_data_analysis.py

```diff
@@ -90,13 +90,24 @@ def test_make_document(self, answers, expected_document):
     )
     @override_settings(
         GROUP_KCS=[
-            'kc_id_1_set-me', 'kc_id_2_set-me', 'kc_id_3_set-me',
-            'kc_id_4_set-me', 'kc_id_5_set-me', 'kc_id_6_set-me',
-            'kc_id_7_set-me'
+            'kc_id_1', 'kc_id_2', 'kc_id_3',
+            'kc_id_4', 'kc_id_5', 'kc_id_6',
+            'kc_id_7'
         ]
     )
     def test_calculate_probabilities(self, answers, expected_probabiities):
+        """
+        Tests that `calculate_probabilities` produces correct dictionary
+        of the following form:
+        {knowledge components:
+        probabilities of user belonging to groups represented by the knowledge components}
+        Assures that the calculated probabilities are correct.
+        """
         probabilities = qda.calculate_probabilities(answers)
+        self.assertEqual(
+            expected_probabiities.keys(),
```

Comment:
@tomaszgy Ack, sorry for not noticing this in the test's signature earlier, but there's a typo here: `probabiities` (missing `l`).

Reply:
@itsjeyd No worries and good catch :) Just corrected.